pkg/operation/botanist/monitoring.go

```diff
@@ -216,6 +222,77 @@ func (b *Botanist) DeploySeedMonitoring(ctx context.Context) error {
 	return nil
 }
 
+func (b *Botanist) getCustomAlertingConfigs(ctx context.Context, alertingSecretKeys []string) (map[string]interface{}, error) {
+	configs := map[string]interface{}{
+		"auth_type": map[string]interface{}{},
+	}
+	// Check if there are custom alerting configurations
+	if len(alertingSecretKeys) > 0 {
+		for _, key := range alertingSecretKeys {
+			secret := b.Secrets[key]
+			if string(secret.Data["auth_type"]) == "none" {
```

Comment:
```suggestion

			if string(secret.Data["auth_type"]) == "none" {
```

Can you make newlines before the `if` blocks to improve readability?